ConwaysGameOfLife/Universe.cs

```diff
@@ -82,6 +78,14 @@ public int CountLiveNeighbours(Cell cell)
             return aliveNeighbours.Count;
         }
 
+        private static void ThrowsExceptionIfCellIsNull(Cell cell)
+        {
+            if (cell == null)
+            {
+                throw new NullReferenceException();
+            }

```

Comment:
have a look into null coalescing if you're interested

```suggestion
            cell ?? throw new NullReferenceException();

```